packages/atomic/src/pages/accessibility/commerce-full.html

```diff
@@ -385,45 +385,47 @@ <h1 class="accessibility-header">Results</h1>
                   margin-right: 0.25rem;
                 }
               </style>
-              <atomic-result-section-badges>
-                <atomic-result-badge field="ec_brand"></atomic-result-badge>
-              </atomic-result-section-badges>
-              <atomic-result-section-visual>
-                <atomic-result-image field="ec_images"></atomic-result-image>
-              </atomic-result-section-visual>
-              <atomic-result-section-title><atomic-result-link></atomic-result-link></atomic-result-section-title>
-              <atomic-result-section-title-metadata>
-                <atomic-result-rating field="ec_rating"></atomic-result-rating>
-                <atomic-result-printable-uri max-number-of-parts="3"></atomic-result-printable-uri>
-              </atomic-result-section-title-metadata>
+              <atomic-result-section-title>
+                <h2><atomic-result-link></atomic-result-link></h2>
+              </atomic-result-section-title>
               <atomic-result-section-emphasized>
                 <atomic-result-number field="ec_price">
                   <atomic-format-currency currency="USD"></atomic-format-currency>
                 </atomic-result-number>
               </atomic-result-section-emphasized>
+              <atomic-result-section-title-metadata>
+                <atomic-result-rating field="ec_rating"></atomic-result-rating>
+                <atomic-result-printable-uri max-number-of-parts="3"></atomic-result-printable-uri>
+              </atomic-result-section-title-metadata>
+              <atomic-result-section-badges>
+                <atomic-result-badge field="ec_brand" role="note"></atomic-result-badge>
+              </atomic-result-section-badges>
+              <atomic-result-section-visual>
+                <atomic-result-image field="ec_images" aria-hidden="true"></atomic-result-image>
+              </atomic-result-section-visual>
               <atomic-result-section-excerpt
                 ><atomic-result-text field="ec_shortdesc"></atomic-result-text
               ></atomic-result-section-excerpt>
-              <atomic-result-section-bottom-metadata>
+              <atomic-result-section-bottom-metadata role="grid">
```

Comment:
Could the role `grid` be coded inside the section? Or would different clients choose different values depending on the structure of the template?

If different, what are the impacts of configuring an incorrect value? e.g. if this role was set to `row` instead of `grid`, what would happen?

Reply:
> Could the role `grid` be coded inside the section? Or would different clients choose different values depending on the structure of the template?

It's possible, I'll move it. Clients are gonna be able to structure things however they want, with some exceptions (such as `atomic-result-fields-list` directly containing the children it can hide).

> If different, what are the impacts of configuring an incorrect value? e.g. if this role was set to `row` instead of `grid`, what would happen?

Screen readers are gonna read results wrong. They may:
* Describe elements inaccurately
* Describe individual parts of an element instead of the element as a whole
* Describe an element as a whole instead of its individual pieces
* Describe an element that should be skipped
* Skip an element that should be described
* Etc.